src/NextGen/Gateways/PayPal/PayPalStandardGateway/payPalStandardGateway.tsx

```diff
@@ -0,0 +1,40 @@
+import type {Gateway} from '@givewp/forms/types';
+import PayPalLogo from "./PayPalLogo";
+
+type PayPalStandardGatewaySettings = {
+    fields: {
+        heading: string;
+        subheading: string;
+        body: string;
+    }
+}
+
+let fields: PayPalStandardGatewaySettings['fields'];
+
+const paypalStandardGateway: Gateway = {
+    id: 'paypal',
+    supportsRecurring: false,
+    supportsCurrency(currency: string): boolean {
+        return false;
+    },
+    initialize() {
+        fields = this.settings.fields;
+    },
+    Fields() {
+        return (
+            <div style={{
+                display: "flex",
+                flexDirection: "column",
+                alignItems: "center",
+                rowGap: "1rem",
+                textAlign: "center"
+            }}>
+                <PayPalLogo/>
+                <b>{fields.heading}</b>
+                <div><b>{fields.subheading}:</b> {fields.body}</div>
+            </div>
+        );
+    }
+}
```

Comment:
This is a great example of a gateway that I think could live entirely in PHP. Whether it supports recurring could be inferred, and we can add a `supportedCurrencies()` method to the gateway which returns an array of supported currencies. Finally, the fields themselves aren't interactive, so the markup could be passed along directly and we use Interweave to display it safely.

Reply:
Yeah I agree, Im thinking things like supportsRecurring and supportsCurrency can be localized from the gateway class so they don't have to add them here manually.  As for using interweave, i'm open to it but want to think through what a clear api for that would look like so gateways are not confused about the difference between JS fields and php rendered fields.